### PR TITLE
Update mandatory info.

### PR DESCRIPTION
**Pull request by BigRoma**

### CODE REVIEW

**Review by Keshoid — APPROVED** [review]
Ok

File: MandatoryDeBotFunctions.md [hunk 1]
```diff
@@ -32,30 +33,33 @@ returns:
...
     
     contract DebotA is Debot {
-    
+        
         function getRequiredInterfaces() override returns (uint256[] interfaces) {
-        return [ID_TERMINAL, ID_MENU, ...];
+            return [ID_TERMINAL, ID_MENU, ...];
+        }
+
+        function getInfo() override returns (string name, string version, string publisher, string key, string author, address support, string hello, string language) {
```
Comment (Keshoid) [inline MandatoryDeBotFunctions.md]:
```suggestion
        function getDebotInfo() public functionID(0xDEB) view override returns(
        string name, string version, string publisher, string caption, string author,
        address support, string hello, string language, string dabi, bytes icon) {
```

File: MandatoryDeBotFunctions.md [hunk 2]
```diff
@@ -14,7 +14,7 @@ returns:
 
     publisher: bytes - utf-8 string with info about who has deployed debot to blokchain, e.g. "TON Labs"
 
-    key: bytes - (10-20 ch.) utf-8 string with short description, e.g. "Work with DePools"
+    caption: bytes - (10-20 ch.) utf-8 string with short description, e.g. "Work with DePools"
```
Comment (sauin) [inline MandatoryDeBotFunctions.md:19]: Debot.sol still has the `key` param

File: MandatoryDeBotFunctions.md [hunk 3]
```diff
@@ -23,6 +23,7 @@ returns:
     hello: bytes - utf-8 string with first messsage with DeBot description, e.g. "Hello, I'm a FreeTON Debot Kapellmeister. I know everything about DePools. Let's start." 
 
     language: bytes - (ISO-639) utf-8 string with debot interface language, e.g. "en"
+
```
Comment (sauin) [inline MandatoryDeBotFunctions.md:28]: Missing `dabi` and `icon` parameters

File: MandatoryDeBotFunctions.md (hunk 2 above)
Comment (BigRoma) [reply MandatoryDeBotFunctions.md:19]: Yep, we will change it after PR merge.

**Review by sauin — APPROVED** [review]

File: MandatoryDeBotFunctions.md [hunk 4]
```diff
@@ -23,6 +25,8 @@ returns:
...
+    icon: bytes - base64 encoded PNG image (144x144 px)
```
Comment (Keshoid) [inline MandatoryDeBotFunctions.md]:
```suggestion
    icon: bytes - base64 encoded PNG image (144x144 px)
```
```suggestion
    icon: bytes - utf8-string with encoded PNG image (144x144 px). The string format is "data:image/png;base64,<image>", where `image` - image bytes encoded in base64. See example below.
```

**Review by Keshoid — APPROVED** [review]

**Review by Keshoid — CHANGES_REQUESTED** [review]
Please, apply suggestions.

**Review by Keshoid — APPROVED** [review]

**Pull request merged**